File: Mobile Applications/android/app/src/main/java/com/example/activityplannerapp/activities/domain/ActivityTypes.kt
package com.example.activityplannerapp.activities.domain

import com.example.activityplannerapp.R

data class ActivityType (
    val name: String,
    val picture: Int
)

object Types {
    val typesList = listOf(
        ActivityType("sport", R.drawable.sports),
        ActivityType("social", R.drawable.social),
        ActivityType("work", R.drawable.work),
        ActivityType("leisure", R.drawable.leisure),
        ActivityType("health", R.drawable.health),
        ActivityType("education", R.drawable.education),
        ActivityType("entertainment", R.drawable.entertainment),
        ActivityType("other", R.drawable.other_stuff)
    )
}

File: Mobile Applications/flutter_app/lib/Domain/ActivityTypes.dart

class ActivityType {
  String name;
  String picture;

  ActivityType(this.name, this.picture);
}

class Types {
  static final List<ActivityType> typesList = [
    ActivityType("sport", "assets/sports.jpg"),
    ActivityType("social", "assets/social.jpg"),
    ActivityType("work", "assets/work.jpg"),
    ActivityType("leisure", "assets/leisure.png"),
    ActivityType("health", "assets/health.png"),
    ActivityType("education", "assets/education.png"),
    ActivityType("entertainment", "assets/entertainment.jpg"),
    ActivityType("other", "assets/other_stuff.png"),
  ];
}